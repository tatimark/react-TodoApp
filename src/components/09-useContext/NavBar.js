import React from 'react';
// import Navbar from "react-bootstrap/Navbar";
import { Link, NavLink } from 'react-router-dom';

export const NavBar = () => {
    return (
        <nav className="navbar navbar-expand-sm navbar-dark bg-dark">
                <Link className="navbar-brand" to="/">useContext</Link>
             
       
                <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
                <div className="navbar-nav">
                    <NavLink  activeClassName="active" className="nav-link" to="/">Home </NavLink>
                    <NavLink activeClassName="active" className="nav-link" to="/about">About</NavLink>
                    <NavLink activeClassName="active" className="nav-link" to="/login">Login</NavLink>
                    
                </div>
                </div>
      </nav>
    )
}
