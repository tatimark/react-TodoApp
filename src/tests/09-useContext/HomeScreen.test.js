import React from 'react';
import { shallow, mount } from 'enzyme';
import { HomeScreen } from '../../components/09-useContext/HomeScreen';
import { UserContext } from '../../components/09-useContext/UserContext';



describe('Pruebas en HomeScreen', () => {

    const user = {
        name:'Tati',
        email: 'tatimark@hotmail.com'
    }

    const wrapper = mount( 
        <UserContext.Provider value={{
            user: user
        }}>
        <HomeScreen />
        </UserContext.Provider>
     );
    test('debe de mostar el componente correctamente ', () => {
        expect( wrapper ).toMatchSnapshot();
    })



})