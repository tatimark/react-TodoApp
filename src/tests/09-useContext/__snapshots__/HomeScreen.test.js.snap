// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HomeScreen debe de mostar el componente correctamente  1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
    "name": "Tati",
    "email": "tatimark@hotmail.com"
}
  </pre>
</div>
`;
